persistence/src/test/java/edu/unc/lib/dl/persist/services/edit/EditTitleServiceTest.java

```diff
@@ -162,21 +183,57 @@ public void multipleTitlesInMods() throws Exception {
         document.addContent(new Element("mods", MODS_V3_NS)
                 .addContent(new Element("titleInfo", MODS_V3_NS)
                         .addContent(new Element("title", MODS_V3_NS).setText("original title")))
-                .addContent(new Element("titleInfo", MODS_V3_NS))
-                        .addContent(new Element("title", MODS_V3_NS).setText("a second title")));
+                .addContent(new Element("titleInfo", MODS_V3_NS)
+                        .addContent(new Element("title", MODS_V3_NS).setText("a second title"))));
         when(binaryObj.getBinaryStream()).thenReturn(convertDocumentToStream(document));
 
         service.editTitle(agent, pid, title);
 
         verify(operationsMessageSender).sendUpdateDescriptionOperation(anyString(), pidsCaptor.capture());
         Collection<PID> pids = pidsCaptor.getValue();
-        assertEquals(pids.size(), 1);
+        assertEquals(1, pids.size());
         assertTrue(pids.contains(pid));
+
+        verify(contentObj).setDescription(inputStreamCaptor.capture());
```

Comment:
you're welcome to put this line into `getUpdatedDescriptionDocument`, but that's just a style thing, its really up to you.